src/menu.tsx

```diff
@@ -86,8 +100,8 @@ export function get_menu_items(
             {
                 id: "paste-item",
                 title: _("Paste"),
-                isDisabled: clipboard.length === 0,
-                onClick: () => pasteFromClipboard(clipboard, cwdInfo, path, addAlert),
+                isDisabled: clipboard.files?.length === 0,
```

Comment:
Shouldn't this also be disabled if clipboard.files is undefined?

Reply:
that should be what is happening:
```
> clipboard.files
undefined
> clipboard.files?.length === 0
false
```